docs(client/ContainerExecAttach): add a mention to stdcopy.StdCopy

PowerPixel opened the pull request:
Add a mention to stdcopy.StdCopy to the documentation, as the stream returned in the HijackedResponse is multiplexed.

Closes #50118 
<!--
Please make sure you've read and understood our contributing guidelines;
https://github.com/moby/moby/blob/master/CONTRIBUTING.md

** Make sure all your commits include a signature generated with `git commit -s` **

For additional information on our contributing process, read our contributing
guide https://docs.docker.com/opensource/code/

If this is a bug fix, make sure your description includes "fixes #xxxx", or
"closes #xxxx"

Please provide the following information:
-->

**- What I did**
I added a mention to `stdcopy.StdCopy` to the documentation of the `client.ContainerExecAttach` method.

**- How I did it**
I simply added a few lines of documentation under the existing one for the `client.ContainerExecAttach` method.

**- How to verify it**

**- Human readable description for the release notes**
<!--
Write a short (one line) summary that describes the changes in this
pull request for inclusion in the changelog.
It must be placed inside the below triple backticks section.

NOTE: Only fill this section if changes introduced in this PR are user-facing.
The PR must have a relevant impact/ label.
-->
```markdown changelog

```

**- A picture of a cute animal (not mandatory but encouraged)**

vvoland reviewed (commented):
Thanks!

vvoland commented on client/container_exec.go:
```diff
@@ -58,6 +58,8 @@ func (cli *Client) ContainerExecStart(ctx context.Context, execID string, config
 // It returns a types.HijackedConnection with the hijacked connection
 // and the a reader to get output. It's up to the called to close
 // the hijacked connection by calling types.HijackedResponse.Close.
+// The Reader contained in the hijacked connection should be read using
+// the stdcopy.StdCopy function.
```
Looks like we have a similar doc on the `ContainerAttach` function:

https://github.com/moby/moby/blob/b6669f1f6428566c94789cedd602ee589fa2e82c/client/container_attach.go#L12-L36

I think we could partially reuse it here too (but spare the technical details and link to `ContainerAttach` for more details).

PowerPixel replied:
@vvoland Hey ! Sorry it tooks me a while to copy those few lines. :p This should be good, I copied the non-technical part of the documentation of `ContainerExec` and added a reference to it in the documentation for those who'd like to dwelve deeper in the technical details.

vvoland reviewed (commented):
Thanks!

One last step: could you please rebase your branch on top of `master` and then squash your commits into one?

PowerPixel committed:
docs(client/ContainerExecAttach): add a mention to stdcopy.StdCopy

Add a mention to stdcopy.StdCopy to the documentation, as the stream returned in the HijackedResponse is multiplexed when tty is disabled.

Signed-off-by: Medhy DOHOU <52136144+PowerPixel@users.noreply.github.com>

PowerPixel commented:
Sorry, took me a while, this should be good now :)

vvoland reviewed (approved):
Thanks!
